### PR TITLE
Document log sanitization feature

### PR DESCRIPTION
**Pull request by serathius**
Documets https://github.com/kubernetes/enhancements/issues/1753

### CODE REVIEW

**Comment by sftim** [comment]
Hi @serathius 

The earlier you can provide (some / most of) the content for documenting this feature, the earlier that SIG Docs can get reviewing. We'd really appreciate a chance to get as much reviewing done as we can before KubeCon.

**Comment by annajung** [comment]
/hold cancel
k/k prs merged

**Comment by SomtochiAma** [comment]
Heyy @serathius,
The docs deadline is on the 23rd of November and we would appreciate it if you could add some content to the PR so we could get started reviewing! Thank you.

**Comment by serathius** [comment]
@SomtochiAma @sftim  PTAL

**Comment by annajung** [comment]
page preview https://deploy-preview-24845--kubernetes-io-vnext-staging.netlify.app/docs/concepts/cluster-administration/system-logs/#log-sanitization

/lgtm

File: content/en/docs/concepts/cluster-administration/system-logs.md [hunk 1]
```diff
@@ -91,6 +91,27 @@ List of components currently supporting JSON format:
...
+{{< feature-state for_k8s_version="v1.20" state="alpha" >}}
+
+{{<warning >}}
+Log sanitization might incur significant computation overhead and therefore should not be enabled in production.
+{{< /warning >}}
+
+The `--experimental-logging-sanitization` controls if a sanitization filter will be installed in klog.
+If enabled all log arguments are inspected for fields tagged as sensitive data (e.g. passwords, keys, tokens) and logging of these fields will be prevented.
+
+List of components currently supporting JSON format:
```
Comment (serathius) [inline content/en/docs/concepts/cluster-administration/system-logs.md:105]:
```suggestion
List of components currently supporting log sanitization:
```

**Comment by sftim** [comment]
@serathius I'm guessing this is good to review? If so, please would you also revise the PR description so that it's completely clear.

**Comment by serathius** [comment]
d

**Review by irvifa — COMMENTED** [review]
/label tide/merge-method-squash
/lgtm

File: content/en/docs/concepts/cluster-administration/system-logs.md [hunk 2]
```diff
@@ -91,6 +91,27 @@ List of components currently supporting JSON format:
...
+
+{{<warning >}}
+Log sanitization might incur significant computation overhead and therefore should not be enabled in production.
+{{< /warning >}}
+
+The `--experimental-logging-sanitization` controls if a sanitization filter will be installed in klog.
+If enabled all log arguments are inspected for fields tagged as sensitive data (e.g. passwords, keys, tokens) and logging of these fields will be prevented.
+
+List of components currently supporting log sanitization:
+* {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
```
Comment (kbhawkey) [inline content/en/docs/concepts/cluster-administration/system-logs.md:106]: nit: Line 102, wording
The `--experimental-logging-sanitization` flag enables the `klog` sanitization filter.

nit: It does not make sense to repeat or overuse the tooltips in this section.
Instead you could list the tools.

nit: Wording in note, Line 111:
The Log sanitization filter does not prevent user workload logs from leaking sensitive data.

**Comment by dashpole** [comment]
/lgtm
from sig-instrumentation

**Review by irvifa — COMMENTED** [review]
I think we can revise later perhaps since it’s already the deadline
/approve

**Comment by kbhawkey** [comment]
👋  @irvifa ,
Sometimes it is good to ask before approving? This is a team effort. Thank you!

**Comment by irvifa** [comment]
@kbhawkey thanks for the reminder will do it next time

**Pull request merged**